waku/factory/external_config.nim

```diff
@@ -312,8 +312,13 @@ type WakuNodeConf* = object
       name: "keep-alive"
     .}: bool
 
+    networkShards* {.
+      desc: "Number of shards in the network", defaultValue: 0, name: "network-shards"
+    .}: uint32
+
     pubsubTopics* {.
-      desc: "Default pubsub topic to subscribe to. Argument may be repeated.",
+      desc:
+        "Deprecated. Default pubsub topic to subscribe to. Argument may be repeated.",
```

Comment:
Let's create a release milestone and take note of the final deprecation. Maybe in `v0.34.0`?

Reply:
We have https://github.com/waku-org/nwaku/issues/2806 open, just changed the milestone to `v0.34.0`

Maybe we shouldn't close this issue after this PR? or create a new one? Any of them works for me